packages/react/src/components/ZoomControl/index.tsx

```diff
@@ -79,7 +82,9 @@ const ZoomControl: React.FC = () => {
       <div
         className="fortune-zoom-button"
         onClick={(e) => {
-          zoomTo(context.zoomRatio - 0.1);
+          if (parseFloat(context.zoomRatio.toFixed(1)) > MIN_ZOOM_RATIO) {
+            zoomTo(parseFloat(context.zoomRatio.toFixed(1)) - 0.1);
+          }
```

Comment:
I mean, you add the condition and `parseFloat` inside `zoomTo`.

And your "Fix scaling value accuracy issues" is wrong.

```js
parseFloat(3.7.toFixed(1)) + 0.1
> 3.8000000000000003
parseFloat((3.7 + 0.1).toFixed(1))
> 3.8
```

Reply:
I understand now. Thank you.

Reply:
Oh, and don't forget to update `zoom.ts` as well